hazelcast/src/main/java/com/hazelcast/internal/networking/nio/iobalancer/LoadTracker.java

```diff
@@ -154,13 +154,17 @@ private void updatePipelineState(MigratablePipeline pipeline) {
         long pipelineLoad = getLoadSinceLastCheck(pipeline);
         pipelineLoadCount.set(pipeline, pipelineLoad);
         NioThread owner = pipeline.owner();
+        if (owner == null) {
```

Comment:
I need to think about this because I hate having exceptional checks on the fast path. This should have been covered somewhere else.

Reply:
ok

Reply:
@pveentjer: this is executed on the I/O Balancer thread, not on the I/O thread.

Reply:
You are right. I didn't inspect the change well enough, I thought it was on the pipeline.process path.